cibuildwheel/macos.py

```diff
@@ -91,7 +91,7 @@ def call(args, env=None, cwd=None, shell=False):
         call(['python', get_pip_script, '--no-setuptools', '--no-wheel'], env=env)
         call(['pip', '--version'], env=env)
         call(['pip', 'install', '--upgrade', 'setuptools'], env=env)
-        call(['pip', 'install', 'wheel'], env=env)
+        call(['pip', 'install', 'git+https://github.com/pypa/wheel.git@master'], env=env)
```

Comment:
Installing from master may not be a good idea, probably better to wait for a release or pin to a specific commit in the mean time

```suggestion
        call(['pip', 'install', 'git+https://github.com/pypa/wheel.git@a7510b6510ab3f1768de7eb75d7a6d06303bee24'], env=env)
```